identity/src/ed25519.rs

```diff
@@ -34,6 +34,7 @@ pub struct Keypair(ed25519::SigningKey);
 
 impl Keypair {
     /// Generate a new random Ed25519 keypair.
+    #[cfg(feature = "rand")]
     pub fn generate() -> Keypair {
```

Comment:
More specifically I am talking about this line. The `rand` feature is `libp2p-identity` specific, correct? Dependents upgrading to `libp2p-identity` `v0.2.5` will not have the `libp2p-identity` `rand` feature enabled no?